### PR TITLE
Pull Request Week 18

### PR DESCRIPTION
**Pull request by claragyokeres**

### CODE REVIEW

**Comment by claragyokeres** [comment]
Pull request for week 18

**Comment by puj** [comment]
Waiting for link

**Comment by claragyokeres** [comment]
https://plantsdatabasetester.herokuapp.com/

File: server.js [hunk 1]
```diff
@@ -27,11 +80,41 @@ const app = express()
...
 app.use(bodyParser.json())
 
+
 // Start defining your routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/plants', async (req, res) => {
+  const plants = await Plants.find().populate('team')
```
Comment (JennieDalgren) [inline server.js]: ⭐️

File: server.js [hunk 2]
```diff
@@ -27,11 +80,41 @@ const app = express()
...
+    res.status(404).json({ error: 'plant not found' })
+  }
+})
+
+app.get('/families/', async (req, res) => {
+  const membersAbove = req.query.members
+
+  const families = membersAbove
+    ? await Family.find({ members: { $gte: membersAbove } })
+    : await Family.find()
```
Comment (JennieDalgren) [inline server.js]: ⭐️ even though memebersAbove is a bad variable name for this?

**Pull request closed without merging**